app/src/main/java/org/oppia/app/options/OptionControlsViewModel.kt

```diff
@@ -81,7 +81,7 @@ class OptionControlsViewModel @Inject constructor(
 
   fun getStoryTextSize(storyTextSize: StoryTextSize): String {
     return when (storyTextSize) {
-      StoryTextSize.SMALL_TEXT_SIZE -> "Small"
+      StoryTextSize.SMALL_TEXT_SIZE_UNSPECIFIED -> "Small"
```

Comment:
Okay these should not change. SMALL_TEXT_SIZE should still exist only that its enum value will become 1
TEXT_SIZE_UNSPECIFIED will be the value 0

Reply:
Got it.